addons/web_editor/static/src/scss/web_editor.variables.scss

```diff
@@ -36,15 +36,18 @@ $o-we-ace-color: #2F3129 !default;
 // Needed to be changed to be high enough to not overflow when a user
 // has a page with a lot of content (10000px was proven to be too small)
 $o-we-handles-offset-to-hide: 100000px !default;
-$o-we-handles-btn-size: 20px !default;
+$o-we-handles-btn-size: 14px !default;
 $o-we-handles-accent-color: $o-we-accent !default;
 $o-we-handles-accent-color-preview: $o-enterprise-color !default;
-$o-we-handle-edge-size: 8px !default;
+$o-we-handle-edge-size: $o-we-handles-btn-size !default;
 $o-we-handle-border-width: 2px !default;
+$o-we-handle-inside-line-width: 3px !default;
 
-$o-we-dropzone-size: 30px !default; // $grid-gutter-width (todo: allow to use the variable)
+$o-we-dropzone-size: 60px !default; // $grid-gutter-width * 2 (todo: allow to use the variable)
```

Comment:
`$grid-gutter-width` is actually `1.5rem (= 24px)` now.
Should this value be `48px` ?

Reply:
In `website` this value is still 30px, I guess we should keep 60px.
https://github.com/odoo/odoo/blob/04eed39aa849aa25248906bd239fa396e90c72c0/addons/website/static/src/scss/bootstrap_overridden.scss#L77C23-L77C23